dub.json

```
@@ -12,7 +12,7 @@
     "StdLoggerDisableWarning"
   ],
   "dependencies" : {
-    "libdparse": ">=0.17.0 <0.18.0",
+    "libdparse": ">=0.17.0 <1.0.0",
```

```suggestion
    "libdparse": ">=0.17.0 <0.18",
```

But I don't see why we're not using path dependencies here ??

---

I saw that @WebFreak001 used 1.0.0 here [1] and @Hackerpilot validated it, so I went with this.

[1] https://github.com/dlang-community/libddoc/pull/54/files#diff-1b617af56e869773e32ce59329f25b2db5d8b3c5151519919c7c10266b4d78fcR23

---

> But I don't see why we're not using path dependencies here ??

Beats me.

---

in ddoc the libdparse version is not as rightly coupled, it's only needed for tokenizing - I think using 1.0.0 here is a bad idea because D-Scanner is much more tightly coupled with the API